### ISSUE TITLE
Crash on screen resize

### ISSUE THREAD

**Issue by T1mL3arn** [Starmapo/flixel-text-input#2]
```haxe
override function create() {

	super.create();

	var address = new flixel.addons.text.FlxTextInput();
	address.multiline = false;
	address.maxChars = 15; // 255.255.255.255
	address.size = 16;
	address.text = '255.255.255.255';
	address.x = 100;
	address.y = 100;
	address.fieldWidth = 200;
	address.fieldHeight = address.height;
	add(address);
}
```
1. test for neko/hashlink `lime test neko -debug`
2. focus textfield with mouse click
3. try to resize window
4. app crashes

```
Uncaught exception: Null access .y
Called from flixel.FlxObject.getScreenPosition(flixel/FlxObject.hx:1097)
Called from flixel.addons.text.FlxTextInput.getPointerPosition(flixel/addons/text/FlxTextInput.hx:97)
Called from flixel.addons.text.FlxTextInput.updateMouseInput(flixel/addons/text/FlxTextInput.hx:129)
Called from flixel.addons.text.FlxTextInput.updateInput(flixel/addons/text/FlxTextInput.hx:61)
Called from flixel.addons.text.internal.FlxBaseTextInput.update(flixel/addons/text/internal/FlxBaseTextInput.hx:356)
Called from flixel.addons.text.FlxTextInput.update(flixel/addons/text/FlxTextInput.hx:47)
Called from flixel.group.FlxTypedGroup.update(flixel/group/FlxGroup.hx:151)
Called from flixel.FlxState.tryUpdate(flixel/FlxState.hx:239)
```

**Comment by T1mL3arn** [comment]
Well, that's me. I have this code to deal with shader distortion

```haxe
Flixel.signals.gameResized.add((_, _)->{
	Flixel.cameras.reset();
	Flixel.camera.filters = cast filters;
	Flixel.camera.filtersEnabled = filtersEnabled;
});
```

And `FlxTextInput` tries to operate on destroyed camera. Let's see if I can fix it.

**Comment by Starmapo** [comment]
This should be fixed by commit 001ffeb, please check if the latest release works for you.

**Comment by T1mL3arn** [comment]
New version works, thanks.
